stable/ambassador/values.yaml

```diff
@@ -46,7 +46,7 @@ securityContext:
 
 image:
   repository: quay.io/datawire/ambassador
-  tag: 0.70.1
+  tag: 0.71.0
```

Comment:
👍 can you uppdate in Chart.yaml as well and README.md? :) (sucks but we need to update in 3 places 😞)

Reply:
I've been meaning to ask about this. Can we include scripting in this repo, or will that break things?

Reply:
Updated